be/src/types/bitmap_value.cpp

```diff
@@ -1154,4 +1154,36 @@ void BitmapValue::add_many(size_t n_args, const uint32_t* vals) {
     }
 }
 
+uint64_t BitmapValueIter::next_batch(uint64_t* values, uint64_t count) {
+    uint64_t remain_rows = _remain_rows();
+    uint64_t read_count = std::min(count, remain_rows);
+    if (read_count <= 0) {
+        return 0;
+    }
+
+    switch (_bitmap->type()) {
+    case BitmapValue::EMPTY:
+        break;
+    case BitmapValue::SINGLE:
+        values[0] = _bitmap->_sv;
+        break;
+    case BitmapValue::SET: {
+        std::vector tmp_array(_bitmap->_set->begin(), _bitmap->_set->end());
+        std::sort(tmp_array.begin(), tmp_array.end());
```

Comment:
every time do this may be expensive.

Reply:
Bitmap output currently requires ordering. The set structure will only be used if the number of elements is less than 32, and the performance overhead is acceptable.